src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```
enum is always in singular